pipelines/rj_crm__disparo_template/utils/dispatch.py

```
@@ -810,53 +932,25 @@ def get_failed_phones(billing_project_id: str) -> set:
-        SELECT targetExternalId
-        FROM ranked
-        WHERE rn = 1
-        AND id_status_disparo = 2
-        -- rn = 1 para pegar o último disparo e id_status_disparo = 2 para selecionar apenas os com falha
+        -- Agora só teremos Salesforce como fonte
+        SELECT DISTINCT cpf
+        FROM `rj-crm-registry.brutos_salesforce.status_disparo`
+        WHERE indicador_quarentena = TRUE
+        AND envio_datahora >= datetime_sub(current_datetime("America/Sao_Paulo"), INTERVAL 4 hour)
+        AND LOWER(nome_hsm) = LOWER('{campaign_name}') if campaign_name else ''
     """
```

Ruff (0.15.21)

[error] Possible SQL injection vector through string-based query construction

(S608)